src/themes/default/components/core/blocks/Auth/Login.vue

```diff
@@ -131,23 +144,11 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-  @import '../../../../css/text.scss';
-
-  input[type=password], input[type=email], input[type=text] {
-    @extend body;
+  input[type=password], input[type=email] {
     box-sizing: border-box;
-    font-size: 18px;
-    padding-top: 10px;
-    padding-bottom: 10px;
-    border: none;
-    border-bottom: 1px solid #BDBDBD;
     width: 100%;
   }
 
-  input[type=submit] {
-    visibility: hidden;
-  }
-
   input::-webkit-input-placeholder {
     color: #BDBDBD;
```

Comment:
this color shouldn't be ahrdcoded, instead injected from colors.scss

Reply:
Done